README.md

```diff
@@ -62,6 +62,50 @@ You can launch the tests as follows:
 dotnet test src/EventStore.sln
 ```
 
+### Build EventStoreDB Docker image
+
+You can also build a Docker image by running the command:
+
+```
+docker build --tag eventstore/eventstore:{version}-{container-runtime} . \
+--build-arg CONTAINER_RUNTIME={container-runtime}
+--build-arg RUNTIME={runtime}
+```
+
+For instance:
+
+```
+docker build --tag eventstore/eventstore:21.10.1-buster-slim . \
+--build-arg CONTAINER_RUNTIME=buster-slim \
+--build-arg RUNTIME=linux-x64
```

Comment:
```suggestion
docker build . --tag eventstore/eventstore:21.10.1-buster-slim . \
--build-arg CONTAINER_RUNTIME=buster-slim \
--build-arg RUNTIME=linux-x64
```

( moved the `.` , to avoid  copy paste error  )
`docker build --tag eventstore/eventstore:21.10.1-buster-slim --build-arg CONTAINER_RUNTIME=buster-slim --build-arg RUNTIME=linux-x64`
outputs 
**_"docker build" requires exactly 1 argument._**